opensrp-chw-core/src/main/java/org/smartregister/chw/core/contract/BaseReferralNotificationDetailsContract.java

```diff
@@ -0,0 +1,22 @@
+package org.smartregister.chw.core.contract;
+
+import org.smartregister.chw.core.domain.ReferralNotificationItem;
+
+public interface BaseReferralNotificationDetailsContract {
+
+    interface View {
+        void onReferralDetailsFetched(ReferralNotificationItem referralNotificationItem);
```

Comment:
Should this be a presenter method instead?

Reply:
This method acted like a callback I have refactored it -> moved it to the presenter then the presenter delegates the task to its corresponding view.